.github/workflows/build.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         php-version: ${{ matrix.php-versions }}
         tools: composer:v2.2
       env:
-        GITHUB_TOKEN: ${{ secrets.TOKEN_GITHUB }}
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
not strictly related to the scope of this PR but reading this [doc](https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication) it seems the default token is enough. I haven't removed the `TOKEN_GITHUB` for now, let's see if we face problems